apps-devstg/us-east-1/backups --/config.tf

```diff
@@ -2,19 +2,18 @@
 # Providers
 #
 provider "aws" {
-  region                  = var.region
-  profile                 = var.profile
-  shared_credentials_file = "~/.aws/${var.project}/config"
+  region  = var.region
+  profile = var.profile
 }
 
 #
 # Backend Config (partial)
 #
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = "~> 1.1.3"
 
   required_providers {
-    aws = "~> 3.2"
+    aws = "~> 4.0"
```

Comment:
@rodriguez-matias Matu remember we've agreed on using `aws  = "~> 4.10"` (https://github.com/binbashar/le-tf-infra-aws/issues/370#issuecomment-1130231254) this is because we found some issues with aws tf provider `>=4.9` in some other layers. Thanks.

Reply:
@exequielrafaela Yes, you are right. I just fixed it with the correct version.